src/components/CreateBuildDialog.tsx

```diff
@@ -96,6 +101,22 @@ class CreateBuildDialog extends React.Component<Props, State> {
 
   render() {
     let { repository } = this.props;
+    if (this.state.error) {
+      const options = {
+        timeout: 5000,
+        position: positions.BOTTOM_CENTER,
+      };
+      this.setState({
+        error: false,
+      });
+      return (
+        <div>
+          <Provider template={AlertTemplate} {...options}>
```

Comment:
Modal alert seems like a confusing UI. A modal alert on top of the modal dialog. 

IMO it will be better to have an `error` string as part of the state and show it right on top or the bottom of the dialog.